1-the-basics/style.css

```diff
@@ -0,0 +1,40 @@
+body {
```

Comment:
prefer classes and avoid styling the <body> tag to prevent side effects. 

```suggestion
.main {
```

Reply:
Yes, I need to correct this bad habit, thank you. 😅